Core/StitchCoreSDK/Sources/StitchCoreSDK/StitchError.swift

```diff
@@ -139,6 +138,9 @@ public enum StitchClientErrorCode: Error {
     case couldNotLoadSyncInfo
     case mongoDriverError(withError: Error)
     case syncRecoveryError(withError: Error)
+    case userNotFound
+    case couldNotFindActiveUser
```

Comment:
[nit] this should be alphabetized, so move this above

Reply:
Ok, this list was not aphetized, but I will change it